seed/static/seed/partials/program_setup.html

```diff
@@ -34,17 +34,17 @@ <h2><i class="fa fa-tachometer"></i> {$:: 'Program Metric Configuration' | trans
                             <li class="r-list-header" translate>Metric</li>
                             <li class="r-row r-row-centered">
                                 <label for="input-metric-name" class="r-no-wrap r-margin-right-5" translate>Name</label>
-                                <input id="input-metric-name" type="text" class="form-control" ng-model="new_compliance_metric.name"/>
+                                <input id="input-metric-name" type="text" class="form-control" ng-model="new_compliance_metric.name" ng-change="program_settings_changed()"/>
                             </li>
                             <li class="r-row r-row-centered">
                                 <label for="input-compliance-period-start" class="r-no-wrap r-margin-right-5" translate>Compliance Period (years)</label>
-                                <input id="input-compliance-period-start" type="number" min=1000 max=9000 class="form-control" ng-model="fields.start_year"/>
+                                <input id="input-compliance-period-start" type="number" min=1000 max=9000 class="form-control" ng-model="fields.start_year" ng-change="program_settings_changed()"/>
                                 <span class="r-margin-left-5 r-margin-right-5">to</span>
-                                <input id="input-compliance-period-end" type="number" min=1000 max=9000 class="form-control" ng-model="fields.end_year"/>
+                                <input id="input-compliance-period-end" type="number" min=1000 max=9000 class="form-control" ng-model="fields.end_year" ng-change="program_settings_changed()"/>
                             </li>
                         </ul>
                         <div class="r-panel r-info r-margin-top-10">
-                            Configure your program metric to enable visualizations on the <b><a ui-sref="insights_program(::{organization_id: org.id})" ui-sref-active="active" translate>program insights</a></b> page.
+                            Configure your program metric to enable visualizations on the <b><a ui-sref="insights_program(::{organization_id: org.id})" ui-sref-active="active" translate>program overview</a></b> page.
```

Comment:
These changes were due to my mistake of accidentally pushing code direct to the 'bps-insights-development' branch instead of a separate branch. I went ahead and removed that commit from the 'bps' branch and add it to my own branch and a PR so we should probably remove these changes in your PR, right?